src/core/adapters/SimplePoolAdapter.ts

```diff
@@ -184,13 +188,16 @@ export abstract class SimplePoolAdapter implements IProtocolAdapter {
             type: TokenType.Protocol,
             tokens: Object.values(underlyingTokenPositions).map(
               (underlyingToken) => {
+                const profit = formatUnits(
+                  profits[underlyingToken.address]!,
+                  underlyingToken.decimals,
+                )
+
                 return {
                   ...underlyingToken,
-                  profitRaw: profits[underlyingToken.address]!,
-                  profit: formatUnits(
-                    profits[underlyingToken.address]!,
-                    underlyingToken.decimals,
-                  ),
+                  profit: isBorrowPosition
+                    ? (-profit).toString()
+                    : profit.toString(),
```

Comment:
```suggestion
                  profit: isBorrowPosition
                    ? (-profit).toString()
                    : profit,
```

`profit` is already a string.